lib/model/addressmanager.js

```diff
@@ -9,19 +12,22 @@ AddressManager.create = function(opts) {
 
   var x = new AddressManager();
 
-  x.version = '1.0.0';
+  x.version = 2;
+  x.derivationStrategy = opts.derivationStrategy || WalletUtils.DERIVATION_STRATEGIES.BIP44;
+  $.checkState(_.contains(_.values(WalletUtils.DERIVATION_STRATEGIES), x.derivationStrategy));
+
   x.receiveAddressIndex = 0;
   x.changeAddressIndex = 0;
-  x.copayerIndex = (opts && _.isNumber(opts.copayerIndex)) ? opts.copayerIndex : SHARED_INDEX;
+  x.copayerIndex = _.isNumber(opts.copayerIndex) ? opts.copayerIndex : BIP45_SHARED_INDEX;
 
   return x;
 };
 
-
 AddressManager.fromObj = function(obj) {
   var x = new AddressManager();
 
   x.version = obj.version;
+  x.derivationStrategy = obj.derivationStrategy || WalletUtils.DERIVATION_STRATEGIES.BIP45;
```

Comment:
I am not 100% sure about this. It seems to go against the rest of the code where BIP45 is the default (for backwards compatibility). Maybe we should default to BIP45 here since the `derivationStrategy` param is always passed?